### PR TITLE
Catroid-499 Send web request _ and store answer in _" brick should be in red Data category of bricks

### PR DESCRIPTION
**Pull request by BeinhAlex**
The WebRequestBrick was removed from the ControlBrickList and got changed to a DataBrick. It was also added to the TestingBrickList. Moved position in the DataBrickList also.

https://jira.catrob.at/browse/CATROID-499

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by wslany** [comment]
> _Please enter a short description of your pull request and add a reference to the Jira ticket._
=> Please follow above instructions.

File: catroid/src/test/java/org/catrobat/catroid/test/content/bricks/CloneBrickUpdateVariableTest.java [hunk 1]
```diff
@@ -69,8 +69,8 @@
 				{"HideTextBrick", new HideTextBrick()},
 				{"ShowTextBrick", new ShowTextBrick()},
 				{"ShowTextColorSizeAlignmentBrick", new ShowTextColorSizeAlignmentBrick()},
-				{"WebRequestBrick", new WebRequestBrick()},
 				{"ReadVariableFromDeviceBrick", new ReadVariableFromDeviceBrick()},
+				{"WebRequestBrick", new WebRequestBrick()},
```
Comment (gPathpp) [inline catroid/src/test/java/org/catrobat/catroid/test/content/bricks/CloneBrickUpdateVariableTest.java]: This change is not necessary but is also not faulty. 
Altogether the code looks nice! 
Eventually, you have to squash the commits, so please undo this change.
